test/test_gdc_postgres_admin.py

```diff
@@ -107,6 +109,7 @@ def test_create_double(self):
 
         self.engine.execute('SELECT * from node_case')
 
+    @pytest.mark.skip(reason="Causing race conditions, so skipping for now.")
```

Comment:
I ran into this and the other one a few times, especially when testing locally. I fixed it in another PR, not sure now if that PR should go in first :thinking: or maybe we can just merge this and that PR will re-enable the test

Reply:
Either way is fine by me.

Reply:
lets get this merged, I can update the other PR